src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      this.result += number;
+
+      return this;
+    },
+    subtract(number) {
+      this.result -= number;
+
+      return this;
```

Comment:
```suggestion

```
There is no need to call these methods in a chain (according to the task requirements)